simulariumio/tests/conftest.py

```diff
@@ -22,8 +22,18 @@
 
 def default_agents_type_mapping() -> Dict[str, Any]:
     return {
-        "0": {"name": "C"},
-        "1": {"name": "U"},
+        "0": {
+            "name": "C",
+            "geometry": {
+                "displayType": "SPHERE",
+            },
+        },
+        "1": {
+            "name": "U",
+            "geometry": {
+                "displayType": "SPHERE",
```

Comment:
is there a reason on the types in the tests are set to defaults? Why not "PDB" for example?

Reply:
OH! this is the expected resulting data, and is your test data never specifying the displayType? In any case, do your tests cover all cases? Not specified, and user specified?

Reply:
yes, some of these are "user" specified and some are defaults, for this one I'm providing some stuff for C, W, S, Y, and L and others are defaults